controllers/snapshot/snapshot_adapter.go

```diff
@@ -76,10 +76,11 @@ func scenariosNamesToList(integrationTestScenarions *[]v1beta1.IntegrationTestSc
 	return &result
 }
 
-// EnsureAllIntegrationTestPipelinesExist is an operation that will ensure that all Integration test pipelines
+// EnsureIntegrationTestPipelinesInRootEnvsExist is an operation that will ensure that all Integration test pipelines
```

Comment:
Didn't we agree to call "Test Pipeline"/"Integration Test PipelineRuns" as just "Integration PipelineRuns"?
```suggestion
// EnsureIntegrationPipelineRunsInRootEnvsExist is an operation that will ensure that all Integration test pipelines
```

Reply:
I'm not aware of that but works for me

Reply:
updated